brewdis-api/src/main/java/com/redislabs/demo/brewdis/RedisSearchUtils.java

```diff
@@ -0,0 +1,135 @@
+package com.redislabs.demo.brewdis;
+
+import com.redislabs.lettusearch.index.IndexInfo;
+import com.redislabs.lettusearch.protocol.CommandKeyword;
+import com.redislabs.lettusearch.search.field.Field;
+import com.redislabs.lettusearch.search.field.TextField;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.redislabs.lettusearch.protocol.CommandKeyword.NOINDEX;
+import static com.redislabs.lettusearch.protocol.CommandKeyword.NOSTEM;
+import static com.redislabs.lettusearch.protocol.CommandKeyword.SORTABLE;
+
+/**
+ * This is copied from here: https://github.com/RediSearch/lettusearch/blob/lettusearch-1.12.1/src/main/java/com/redislabs/lettusearch/RediSearchUtils.java
+ * And changed these:
+ * 1. Change class name from "RediSearchUtils" to "RedisSearchUtils".
+ * 2. Other changes are commented in code. It's not finished yet, I'm still working on it.
+ */
+public class RedisSearchUtils {
```

Comment:
FYI: [lettusearch](https://github.com/RediSearch/lettusearch) is archived, I'll try to use [lettucemod](https://github.com/redis-developer/lettucemod) instead.